mundane-io/src/test/scala/com/ambiata/mundane/io/LocalPathSpec.scala

```diff
@@ -56,6 +56,12 @@ class LocalPathSpec extends Specification with ScalaCheck with DisjunctionMatche
 
     ${ prop((p: Path) => LocalPath.fromFile(p.toFile).map(_.path) must beOkValue(p)) }
 
+  LocalPath.fromURI can extract local paths from URI's
+
+    ${ prop((p: Path) => p != Relative ==> { LocalPath.fromURI(new java.net.URI(s"file:${p.path}")) ==== LocalPath(p).some }) }
```

Comment:
I can't create a valid `URI` with just `Relative` because its and empty string when rendering it.